Frameworks/CoreGraphics/CGColorSpace.mm

```diff
@@ -38,135 +39,161 @@
 const CFStringRef kCGColorSpaceITUR_2020 = static_cast<CFStringRef>(@"kCGColorSpaceITUR_2020");
 const CFStringRef kCGColorSpaceROMMRGB = static_cast<CFStringRef>(@"kCGColorSpaceROMMRGB");
 
-static IWLazyClassLookup _LazyUIColor2("UIColor");
+static CGColorSpaceRef s_rgbColorSpace;
```

Comment:
these should perhaps be StrongCF; that'll make sure they are "owned" by someone.